test/checks/label/multiple-label.js

```diff
@@ -176,4 +176,23 @@ describe('multiple-label', function() {
 		var target = fixture.querySelector('#Q');
 		assert.isTrue(checks['multiple-label'].evaluate.call(checkContext, target));
 	});
+
+	(shadowSupported ? it : xit)(
+		'should consider labels in the same document/shadow tree',
+		function() {
+			fixture.innerHTML = '<div id="target"></div>';
+
+			var target = document.querySelector('#target');
+			var shadowRoot = target.attachShadow({ mode: 'open' });
+			shadowRoot.innerHTML =
+				'<input id="myinput"></input><label for="myinput">normal</label>';
+			var shadowTarget = target.shadowRoot;
+			assert.isFalse(
```

Comment:
Add a second label, and test that this returns true. Right now, this test only proves that there weren't two or more labels anywhere in the DOM.